.lagoon.yml

```diff
@@ -15,7 +15,8 @@ tasks:
   post-rollout:
     - run:
         name: Show DrevOps variables.
-        command: env -0  | sort -z | tr '\0' '\n' | grep ^DREVOPS_ || true
+        command: |
+          [ "${DREVOPS_DEBUG-}" = "1" ] && env -0  | sort -z | tr '\0' '\n' | grep ^DREVOPS_ || true
```

Comment:
Is `${DREVOPS_DEBUG-}` the right syntax?

Reply:
yeah, this is a shortened version of `"${DREVOPS_DEBUG:-}"`